src/WorldStorage/WSSAnvil.cpp

```diff
@@ -2662,15 +2664,34 @@ void cWSSAnvil::LoadSilverfishFromNBT(cEntityList & a_Entities, const cParsedNBT
 
 void cWSSAnvil::LoadSkeletonFromNBT(cEntityList & a_Entities, const cParsedNBT & a_NBT, int a_TagIdx)
 {
+	// Wither skeleton is a separate mob in Minecraft 1.11+, but
+	// we need this to load them from older worlds
```

Comment:
This comment doesn't explain sufficiently why there's a witherskeleton in skeleton loading code. Add `where wither skeletons were only a skeletons with a flag`